.github/workflows/build.yml

```diff
@@ -266,3 +266,56 @@ jobs:
       with:
         name: integration-test-results-db2-${{ matrix.java }}
         path: integration-test-results
+  db2-migration:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 'openjdk8' ]
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up OpenJDK
+      uses: joschi/setup-jdk@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - name: Build samples
+      run: mvn -B install --file fhir-examples --no-transfer-progress
+    - name: Build parent without tests
+      run: |
+        mvn -B install --file fhir-parent -DskipTests -P integration --no-transfer-progress
+    - name: Server Integration Tests
+      env:
+        # debian-based linux uses C.UTF-8 by default and Derby doesn't like that
+        LC_ALL: en_US.UTF-8
+      run: |
+        export WORKSPACE=${GITHUB_WORKSPACE}
+        build/db2-migration-test-docker.sh
+        mvn -B test -DskipTests=false -f fhir-server-test -DskipWebSocketTest=true --no-transfer-progress
+        build/post-integration-test-docker.sh
+    - name: Gather error logs
+      if: failure()
+      run: |
+        it_results=integration-test-results
+        rm -fr ${it_results} 2>/dev/null
+        mkdir -p ${it_results}/server-logs
+        mkdir -p ${it_results}/fhir-server-test
+        containerId=$(docker ps -a | grep fhir | cut -d ' ' -f 1)
+        if [[ -z "${containerId}" ]]; then
+            echo "Warning: Could not find fhir container!!!"
+        else
+            echo "fhir container id: $containerId"
+
+            # Grab the container's console log
+            docker logs $containerId  >& ${it_results}/docker-console.txt
+
+            echo "Gathering post-test server logs from docker container: $containerId"
+            docker cp -L $containerId:/logs ${it_results}/server-logs
+        fi
+        echo "Gathering integration test output"
+        cp -pr ${GITHUB_WORKSPACE}/fhir-server-test/target/surefire-reports/* ${it_results}/fhir-server-test
+    - name: Upload logs
+      if: always()
+      uses: actions/upload-artifact@master
+      with:
+        name: db2-migration-test-results-${{ matrix.java }}
+        path: integration-test-results
```

Comment:
are you going to make db2 migration test part of the pipeline? seems it's only needed if there is any new db schema version available ...  and are you going to cover the migration path from all the previous versions to the current version ...

Reply:
for now, I just wanted to get db2 migration tests into the pipeline...we can add smarts about selectively running it in the future

Reply:
without this, then there is no way to know if the migrations you wrote actually work with db2

Reply:
at a minimum the code would have to run with any downstream dependency changes (database-utils/model)